projects/pluto/src/main/resources/application.yml

```diff
@@ -45,10 +45,10 @@ logging:
 
 security:
   oidc:
-    auth-url: ${pluto.oauth2.base-url}/realms/${pluto.oauth2.realm}/protocol/openid-connect/auth
-    logout-url: ${pluto.oauth2.base-url}/realms/${pluto.oauth2.realm}/protocol/openid-connect/logout?redirect_uri=%s
-    token-url: ${pluto.oauth2.backend-url}/realms/${pluto.oauth2.realm}/protocol/openid-connect/token
-    jwk-key-set-url: ${pluto.oauth2.backend-url}/realms/${pluto.oauth2.realm}/protocol/openid-connect/certs
+    auth-url: ${pluto.oauth2.base-url}/auth/realms/${pluto.oauth2.realm}/protocol/openid-connect/auth
+    logout-url: ${pluto.oauth2.base-url}/auth/realms/${pluto.oauth2.realm}/protocol/openid-connect/logout?redirect_uri=%s
+    token-url: ${pluto.oauth2.backend-url}/auth/realms/${pluto.oauth2.realm}/protocol/openid-connect/token
+    jwk-key-set-url: ${pluto.oauth2.backend-url}/auth/realms/${pluto.oauth2.realm}/protocol/openid-connect/certs
```

Comment:
the "/auth" part was removed from the OIDC URL in Keycloak 18. For the setup we have for fns and KEycloak 20 removing `/auth` part was necessary... Why this change to add it back?

Reply:
I added it back because otherwise I receive `Exception in thread "main" java.lang.reflect.InvocationTargetException` exceptions, causing the fairspace-keycloak container to repeatedly close and restart again.

Reply:
you're right, also for me locally it still requires /auth... That is strange, because according to the documentation it shouldn't be the case. 
Anyway we need to enable setting these parameters without /auth, otherwise fns won't work... 
Let's discuss tomorrow